tools/freebayes/freebayes.xml

```diff
@@ -80,7 +80,11 @@
     #elif str( $options_type.options_type_selector ) == "full":
  
         #if $options_type.optional_inputs.optional_inputs_selector:
-        
+       
+	  #if $options_type.optional_inputs.report_monomorphic
+	    ${options_type.optional_inputs.report_monomorphic}
+	  #end if
```

Comment:
You don't need the `#if ...` and `#end if`, just the middle line as it was before.